subprojects/gradle/instrumentation-tests-default-config/src/main/kotlin/com/avito/instrumentation/InstrumentationDefaultConfigPlugin.kt

```diff
@@ -210,6 +225,41 @@ class InstrumentationDefaultConfigPlugin : Plugin<Project> {
         }
     }
 
+    private fun registerModifiedUiConfig(
```

Comment:
@dsvoronin Should I reduce the amount of the boilerplate? I stopped by thinking you are moving those pieces to avito

Reply:
nope, sadly we'll drop all of these